hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdiff/RocksDBCheckpointDiffer.java

```diff
@@ -362,6 +350,12 @@ private AbstractEventListener newCompactionBeginListener() {
       public void onCompactionBegin(RocksDB db,
           CompactionJobInfo compactionJobInfo) {
 
+        // TODO: Skip (return) if no snapshot has been taken yet
+
+        // Note the current compaction listener implementation does not
+        // differentiate which column family each SST store. It is tracking
+        // all SST files.
```

Comment:
It would be good to create a jira ticket for this, to track as a possible optimization.

Reply:
Filed HDDS-7559